lib/linguist/languages.yml

```diff
@@ -3738,12 +3738,12 @@ Ruby:
   - Deliverfile
   - Fastfile
   - Gemfile
-  - Gemfile.lock
```

Comment:
I think it makes sense to leave Gemfile.lock alone here, but otherwise I'd ✅  the rest of these changes related to `Rakefile` and `.irbrc`.